unattended_installer/install_functions/checks.sh

```diff
@@ -6,6 +6,22 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
+systems_list=("centos","ubuntu","debian","fedora","amzn","rhel","opensuse-leap", "opensuse-tumbleweed","sles","ol","rocky","almalinux")
```

Comment:
This list must be updated with: https://opensearch.org/docs/latest/opensearch/install/compatibility/

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/1222/commits/32e02871a05c29f90715359ed018dce058aad542